packages/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTInstance.mm

```diff
@@ -490,4 +516,8 @@ - (void)_handleJSErrorMap:(facebook::react::MapBuffer)errorMap
                      isFatal:errorMap.getBool(JSErrorHandlerKey::kIsFatal)];
 }
 
+- (void)didReceiveReloadCommand { 
+  [self _loadJSBundle:[self->_bridgeModuleDecorator.bundleManager bundleURL]];
+}
+
```

Comment:
@okwasniewski, why did you end up registering the RCTInstance as a reload command listener here?

Reply:
When clicking Reload on the redbox nothing was happening